Doc/c-api/structures.rst

```diff
@@ -626,6 +626,25 @@ Macro name                       C type                        Python type
 .. c:macro:: Py_T_ULONG          :c:expr:`unsigned long`       :py:class:`int`
 .. c:macro:: Py_T_ULONGLONG      :c:expr:`unsigned long long`  :py:class:`int`
 .. c:macro:: Py_T_PYSSIZET       :c:expr:`Py_ssize_t`          :py:class:`int`
+.. c:macro:: Py_T_SSIZE          :c:expr:`Py_ssize_t`          :py:class:`int`
+.. c:macro:: Py_T_SIZE           :c:expr:`size_t`              :py:class:`int`
+.. c:macro:: Py_T_INT8           :c:expr:`int8_t`              :py:class:`int`
+.. c:macro:: Py_T_UINT8          :c:expr:`uint8_t`             :py:class:`int`
+.. c:macro:: Py_T_INT16          :c:expr:`int16_t`             :py:class:`int`
+.. c:macro:: Py_T_UINT16         :c:expr:`uint16_t`            :py:class:`int`
+.. c:macro:: Py_T_INT32          :c:expr:`int32_t`             :py:class:`int`
+.. c:macro:: Py_T_UINT32         :c:expr:`uint32_t`            :py:class:`int`
+.. c:macro:: Py_T_INT64          :c:expr:`int64_t`             :py:class:`int`
+.. c:macro:: Py_T_UINT64         :c:expr:`uint64_t`            :py:class:`int`
+.. c:macro:: Py_T_INTMAX         :c:expr:`intmax_t`            :py:class:`int`
+.. c:macro:: Py_T_UINTMAX        :c:expr:`uintmax_t`           :py:class:`int`
```

Comment:
I would prefer to not add `Py_T_INTMAX` and `Py_T_UINTMAX` now. They are not used in Python right now. We can add them later if needed.

Reply:
Agree, it is unlikely that they are used for fields in some API.